stories/documentation/navigation/vertical-navigation/vertical-navigation-basic.stories.ts

```diff
@@ -16,36 +16,44 @@ export default {
 } as Meta;
 
 function getTemplate(args: VerticalNavigationBasicStory): string {
-	const disabled = args.disabled ? `disabled` : '';
-	const disabledLink = args.disabled ? `is-disabled` : '';
-	const disabledLinkTabIndex = args.disabled ? `tabindex="-1"` : '';
+	const disabled = args.disabled ? ` is-disabled` : '';
+	const disabledTabIndex = args.disabled ? ` tabindex="-1"` : '';
 	return `<nav class="verticalNavigation">
+	<h3 class="verticalNavigation-sectionTitle">Section title</h3>
 	<ul class="verticalNavigation-list">
 		<li class="verticalNavigation-list-item">
-			<button class="verticalNavigation-list-item-link" aria-expanded="false" ${disabled}>Item <span aria-hidden="true" class="lucca-icon icon-southArrow"></span></button>
+			<button class="verticalNavigation-list-item-link" aria-expanded="false"><span aria-hidden="true" class="verticalNavigation-list-item-link-icon lucca-icon icon-heart"></span>Item<span aria-hidden="true" class="lucca-icon icon-southArrow verticalNavigation-list-item-link-arrow"></span></button>
 			<ul class="verticalNavigation-list mod-child">
 				<li class="verticalNavigation-list-item">
 					<a href="#" class="verticalNavigation-list-item-link">Item</a>
-				</li>
-				<li class="verticalNavigation-list-item">
+					</li>
+					<li class="verticalNavigation-list-item">
 					<a href="#" class="verticalNavigation-list-item-link">Item</a>
-				</li>
-			</ul>
-		</li>
+					</li>
+					</ul>
+					</li>
 		<li class="verticalNavigation-list-item">
-			<button class="verticalNavigation-list-item-link" aria-expanded="true">Item <span aria-hidden="true" class="lucca-icon icon-southArrow"></span></button>
+			<button class="verticalNavigation-list-item-link" aria-expanded="true"><span aria-hidden="true" class="verticalNavigation-list-item-link-icon lucca-icon icon-heart"></span>Item<span aria-hidden="true" class="lucca-icon icon-southArrow verticalNavigation-list-item-link-arrow"></span></button>
 			<ul class="verticalNavigation-list mod-child">
 				<li class="verticalNavigation-list-item">
-					<a href="#" class="verticalNavigation-list-item-link ${disabledLink}" ${disabledLinkTabIndex}>Item</a>
+					<a href="#" class="verticalNavigation-list-item-link${disabled}"${disabledTabIndex}>Item</a>
```

Comment:
What does `${disabled}` do? It's added as a class here. I would use a `span` rather than an unfocusable link.